addons/mail/static/src/models/thread_cache/thread_cache.js

```diff
@@ -444,19 +466,7 @@ function factory(dependencies) {
          * computed and should be considered read-only.
```

Comment:
remove 2nd sentence (confusing)

Reply:
@alexkuhn Done. Also added doc for `_isInUpdateCycle`